modules/examples/filterx/example-filterx-func/example-filterx-func-plugin.c

```diff
@@ -29,7 +29,7 @@
 #include "filterx/object-primitive.h"
 
 static FilterXObject *
-echo(GPtrArray *args)
+echo(FilterXExpr *s, GPtrArray *args)
```

Comment:
I think this change should be in `filterx: add filterx error handling to simple functions argument validation`.

Reply:
I have fixed this myself.